rasa_nlu/classifiers/mitie_intent_classifier.py

```diff
@@ -45,16 +45,25 @@ def train(self, training_data, mitie_file, num_threads):
         for example in training_data.intent_examples:
             tokens = mitie.tokenize(example["text"])
             trainer.add_labeled_text(tokens, example["intent"])
-        self.clf = trainer.train()
+
+        if training_data.intent_examples:
+            # we can not call train if there are no examples!
+            self.clf = trainer.train()
 
     def process(self, tokens, mitie_feature_extractor):
         # type: (List[Text], mitie.total_word_feature_extractor) -> Dict[Text, Any]
 
-        intent, score = self.clf(tokens, mitie_feature_extractor)
+        if self.clf:
+            intent, confidence = self.clf(tokens, mitie_feature_extractor)
+        else:
+            # either the model didn't get trained or it wasn't provided with any data
```

Comment:
Would it make sense to log a warning for this somewhere, once and not on every processed item?